File: TT4/t3.java
import java.util.Scanner;
public class t3{
    private static int number = 0, letter = 0, other = 0;
    public static void main(String[] args){
        Scanner scan = new Scanner(System.in);
        String str = scan.nextLine();
        for (int i = 0; i < str.length(); i++)
            checkCharacter(str.charAt(i));
        System.out.printf("letter: %d numbers: %d other: %d\n", letter, number, other);
    }
    private static void checkCharacter(char a){
        if (('A' <= a && 'Z' >= a) || ('a' <= a && a <= 'z'))
            letter++;
        else if ('0' <= a && '9' >= a)
            number++;
        else
            other++;
    }
}

File: TT4/t3.cpp
#include <stdio.h>
void checkCharacter(char a);
int number = 0, letter = 0, other = 0;
int main() {
	char a;
	do
	{
		a = getchar();
		checkCharacter(a);
	} while (a != '\n');
	other--;
	printf("letter: %d numbers: %d other: %d\n", letter, number, other);
	return 0;
}
void checkCharacter(char a) {
	if (('A' <= a && 'Z' >= a) || ('a' <= a && a <= 'z'))
		letter++;
	else if ('0' <= a && '9' >= a)
		number++;
	else
		other++;
}